src/display.cpp

```diff
@@ -1775,14 +1775,20 @@ void display::enable_menu(const std::string& item, bool enable)
 	}
 }
 
-void display::announce(const std::string& message, const color_t& color, int lifetime)
+void display::announce(const std::string& message, const color_t& color
+                       , const announce_options& options)
 {
-	font::remove_floating_label(prevLabel);
+	if (message.find("nem") != std::string::npos || message.find("keep") != std::string::npos) {  //   FIXME Artificiously add some code to serve as breakpoint beacon, remove before merging.
```

Comment:
...what does "artificiously" mean?

Reply:
I'm sorry (x3).

* I slipped a typo and typed 'server' in a comment in other lines while I was wanting to type 'serve'. I suspect of Xcode first, then of me second.

* I invented this sort of spanish language false friend 'artificiously', I would rather better say... maybe, just _force_. So I _forced_ that line into this commit, where it doesn't belong, as in telling you _act like if it wasn't really there_. And typed the reason: it is a debugger breakpoint beacon, as in _you want to break there only on certain condition met_, that condition is that `if` guard. Being portable between platforms. So onto bullet three...

* Now I realize it would have been better to start with a `DO NOT MERGE.` commit introducing those garbage lines, then the meaningful pull request source diff, finally a third `Thrash garbage.` commit reverting the first one. Once approved, if that where the case, my work would be just remove both commits and force push, instead of before forcing push amending to clear the rubbish, with definitely greater chance of messing and breakage of meaningful lines.

So I'm sorry (x3).

I'll be forcing push of the commit into three commits as per reasoned.

Thank you!

Reply:
Random useless aside (feel free to skip if language is not a topic of interest):

"artificiously" could presumably be analyzed as "artifice" + "-ous" + "-ly", thus meaning "in a manner possessing artifice". As for what "artifice" means, well, I'll leave the [wiktionary](https://en.wiktionary.org/wiki/artifice#Noun) link here, but I'm pretty sure that wasn't the intent. ;)

(Yes, it's totally possible to use a dictionary to find the meaning of words that aren't in the dictionary!)

Then again, there's also [this](https://en.wiktionary.org/wiki/artificious). By that definition, "artificiously" would mean "in an artificial manner", ie "artificially".